spec/integration/transport/winrm_spec.rb

```diff
@@ -180,14 +179,7 @@ def stub_winrm_to_raise(klass, message)
       target.options.delete('cacert')
       target.options['ssl-verify'] = false
 
-      expect(winrm.run_command(omi_target, command)['stdout']).to eq("#{user}\r\n")
```

Comment:
Is there any value in keeping the kerberos tests/infra definitions around for local testing, but not run in CI?

Reply:
I figured I would leave them to document the current state of the system at least.

This test in particular was removed because it's... just wrong. It disables SSL and then tries to connect to the HTTPS port and expects that that will work. I'm unclear how it worked before unless OMI silently downgrades??